metaseq/checkpoint_utils.py

```diff
@@ -179,14 +179,41 @@ def _delete_old_checkpoint_files(
                 os.remove(old_chk)
 
 
+def verify_shards(cfg, dir=None, checkpoint_name=None):
+    # verifies that all the shards of the checkpoint are present
+    checkpoint_name = checkpoint_name.replace(".pt", "")
+    num_gpus = DistributedTrainingConfig.distributed_world_size
```

Comment:
You should get the distributed_world_size from cfg instead... Is it not in cfg?

Reply:
unfortunately, it is not in cfg
![image](https://user-images.githubusercontent.com/13079247/177079516-2ba711e7-38a2-401d-8a76-60420eb0e454.png)